components/ProductsList/index.js

```diff
@@ -1,11 +1,15 @@
 import ProductListItem from "../ProductListItem";
-export default function ProductsList({ products }) {
+export default function ProductsList({ products, handleDeleteProduct }) {
   return (
     <>
       <h2>Product List</h2>
       <ul>
```

Comment:
It might be worth considering checking if there are products to loop over. 

Otherwise you could potentially end up with an empty ul

`<ul></ul>`

Reply:
Good point. Thanks. We added a message the the user, when the product list is empty